src/Components/PaperEditor/TranscriptsContainer/TranscriptTabContent/TranscriptMenu.js

```diff
@@ -34,38 +32,33 @@ const TranscriptMenu = (props) => {
       variant="outline-secondary"
     >
       <LabelsList
-        isLabelsListOpen={ isLabelsListOpen }
-        labels={
-          labels && labels
-        }
+        labels={ labels && labels }
```

Comment:
I've never understood the `labels && labels` - what is this doing?

Reply:
checking if it exists and assigning I think :) 

Verified: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators